js_modules/dagster-ui/packages/ui-core/src/instance/backfill/BackfillRow.tsx

```diff
@@ -75,9 +75,7 @@ export const BackfillRowLoader = (props: {
   const {data} = statusQueryResult;
   const {hasCancelableRuns} = React.useMemo(() => {
     if (data?.partitionBackfillOrError.__typename === 'PartitionBackfill') {
-      if ('partitionBackfill' in data.partitionBackfillOrError) {
```

Comment:
just confirming, this line was causing hasCancelableRuns to never be true, since partitionBackfill is not a field in hte underlying graphql query? Trying to figure out why this would have been added in the first place